dracula-theme.el

```diff
@@ -56,7 +55,8 @@
    'dracula
    ;; default
    `(cursor ((,class (:background ,fg3))))
-   `(default ((,class (:background ,bg1 :foreground ,fg1))))
+   `(default ((((type nil)) (:background "#000000" :foreground ,fg1))
```

Comment:
I like the approach of [using type nil](https://www.gnu.org/software/emacs/manual/html_node/elisp/Window-Systems.html#Window-Systems) but documentation around `default` is kind of vague. Will having multiple defaults like this mean that if type is nil, then only the background of `#000000` will be set, otherwise we'll use the standard `bg1` (aka.. if/else)? Or will this apply both but type nil has a different default? I'm just trying to understand how this code is evaluated.